dev-packages/e2e-tests/test-applications/solidjs/tests/performance.test.ts

```diff
@@ -0,0 +1,102 @@
+import { expect, test } from '@playwright/test';
+import { waitForTransaction } from '@sentry-internal/test-utils';
+
+test('sends a pageload transaction', async ({ page }) => {
+  const transactionPromise = waitForTransaction('solidjs', async transactionEvent => {
+    return !!transactionEvent?.transaction && transactionEvent.contexts?.trace?.op === 'pageload';
+  });
+
+  const [, pageloadTransaction] = await Promise.all([page.goto('/'), transactionPromise]);
+
+  expect(pageloadTransaction).toMatchObject({
+    contexts: {
+      trace: {
+        op: 'pageload',
+        origin: 'auto.pageload.browser',
+      },
+    },
+    transaction: '/',
+    transaction_info: {
+      source: 'url',
+    },
+  });
+});
+
+test('sends a navigation transaction', async ({ page }) => {
+  page.on('console', msg => console.log(msg.text()));
+  const pageloadTxnPromise = waitForTransaction('solidjs', async transactionEvent => {
```

Comment:
Actually no need to wait on this, we can just wait for the navigation only, that should be all fine! Just go to `/` and then navigate, should always emit a navgiation span either way.

Reply:
Right, removed.